### DISCUSSION TITLE
% as #/$

### DISCUSSION THREAD

**Discussion by Superninja9919** [termux/termux-packages#11535] [Q&A]
I have a problem my termux open up with % sign
![Screenshot_20220811_110146_com termux](https://user-images.githubusercontent.com/110962533/184100280-6091d9f0-a1aa-43c2-9166-a46ce423e8f5.jpg)

**Comment by sylirre** [comment]
Try `chsh -s bash` and then restart Termux app.

Looks like you set `zsh` as default shell. It uses `%` as prompt.
